### PR TITLE
Confdbv2 task addition

### PR DESCRIPTION
**Pull request by bsataric**
This PR has been made in order to add the tasks functionallity into the ConfDB.

Functionality of tasks is very similar to one of the Sequences.

Long description of the task object is given here: 

https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideAboutPythonConfigFile#Task_Objects

Current setup is not able to connect to production database and uses developement database as code is still in testing phase.

### CODE REVIEW

File: src/confdb/db/ConfDB.java [hunk 1]
```diff
@@ -1182,25 +1233,27 @@ private void loadConfiguration(Configuration config) throws DatabaseException {
...
+			/*
+			 * if (pera) return;
+			 */
+
 			psSelectPathEntries.setInt(1, configId);
 			psSelectPathEntries.setInt(2, configId);
 			// System.err.println("Trying Pathentries"+configId);
 			rsPathEntries = psSelectPathEntries.executeQuery();
 
-			// This is to fix the operator field bug for modules inside sequences. bug
```
Comment (fwyzard) [inline src/confdb/db/ConfDB.java:1191]: what happens to his part ?

Comment (bsataric) [reply src/confdb/db/ConfDB.java:1191]: You mean why is the code commented below rsPathEntries = psSelectPathEntries.executeQuery(); deleted?

Comment (fwyzard) [reply src/confdb/db/ConfDB.java:1191]: Ah, right, I had seen the comment but han't realised the code below it was already commented out.

**Comment by Sam-Harper** [comment]
closing PR as it has been superseeded

**Pull request closed without merging**